src/main/java/com/minecolonies/coremod/colony/buildings/BuildingDeliveryman.java

```diff
@@ -95,6 +88,13 @@ public void writeToNBT(@NotNull final NBTTagCompound compound)
         super.writeToNBT(compound);
     }
 
+    @NotNull
+    @Override
+    public String getJobName()
+    {
+        return DELIVERYMAN;
+    }
+
     @Override
     public void serializeToView(@NotNull final ByteBuf buf)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this method to simply inherit it. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1185)